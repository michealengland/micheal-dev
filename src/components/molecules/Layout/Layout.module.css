.layout {
  @apply bg-nuetral-white;
}
