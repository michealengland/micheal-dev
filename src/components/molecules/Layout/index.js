import PropTypes from 'prop-types'
import React from 'react'
import styles from './Layout.module.css'
import Header from '@components/organisms/Header'
import Footer from '@components/organisms/Footer'

const Layout = ({children}) => {
  return (
    <div id="layout" className={styles.layout}>
      <Header />
      {children}
      <Footer />
    </div>
  )
}

export default Layout

Layout.propTypes = {
  children: PropTypes.object,
}
