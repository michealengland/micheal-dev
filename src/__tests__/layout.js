import React from 'react'
import renderer from 'react-test-renderer'
import Layout from '../components/molecules/Layout'
describe('Layout', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Layout />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
