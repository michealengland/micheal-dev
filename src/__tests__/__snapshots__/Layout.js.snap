// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div
  className="layout"
  id="layout"
>
  <header
    className="header"
  >
    <div
      className="logo"
    >
      <a
        displayType={false}
        href="/"
      >
        <img
          alt="Micheal.dev logo"
          src="test-file-stub"
        />
      </a>
    </div>
    <nav
      className="menu"
    >
      <ul>
        <li>
          <a
            displayType={false}
            href="/blog"
          >
            Blog
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <h1>
    Some content...
  </h1>
</div>
`;
